composer.json

```diff
@@ -28,13 +28,13 @@
         "source": "https://github.com/cvo-technologies/cakephp-gearman"
     },
     "require": {
+        "php": ">=5.5.9",
         "ext-gearman": "*",
         "cakephp/cakephp": "~3.2"
     },
     "require-dev": {
-        "cakephp/cakephp-codesniffer": "2.*",
```

Comment:
I basically follow what muffin is doing and they don't include it in any of their plugins. Could be because it is already a dependency in cakephp but not sure.